File: src/main/java/com/brian/Hello.java
package com.brian;

public class Hello {
    public static void main(String[] args) {
//        System.out.println("Hello World");
//        new Person().hello();
//        String s = new String("dog");
        String s = "airplane";
        System.out.println(s.charAt(3));
        System.out.println(s.substring(1,4));


        Person p = new Person("Brian", 66.5f, 1.7f);
        p.hello();
        System.out.println(p.bmi());
        int score = 88;
        System.out.println(score < 80 || score > 85);
        char c = 'A';
        System.out.println(c > 'a');

        /*int age = 19;
        char c = 'A';
        byte b = 120;
        float weight = 66.5f;
        float height = 1.7f;
        boolean adult = true;
        boolean enroll = false;
        // Reference data type
        String name = "Brian";
        Integer age2 = 19;
        Character c2 = 'A';*/
    }

}

File: src/main/kotlin/com/brian/Hello.kt
package com.brian

fun main() {
//    println("Hello Kotlin")
//    Human().hello()
//    h.hello()
    var s:String? = "airplane"
//    s = null
//    println(s!!.length)
    println(s?.get(3))
    println(s?.substring(3))

    val h = Human(weight = 66.5f, height =  1.7f)
    println(h.bmi())
    val score = 88
    print(score > 60)
    var c: Char = 'a'
    println(c.toInt() >60)


    /*var age : Int = 19
    age = 20
    var weight = 66.5f
    var name : String*/


}

class Human(name: String="", var weight: Float, var height: Float) {
    init {
        println("test $weight")
    }
//    constructor(name: String, weight: Float, height: Float): this(weight, height)
    fun bmi(): Float {
        val bmi = weight / (height*height)
        return bmi
    }
    fun hello() {
        println("Hello Kotlin")
    }

}